### PR TITLE
Removing dependence on git in gemspec

### LINKED ISSUE #235: Remove git dependence in gemspec
Currently the gemspec has a dependency on git by using the git ls-files syntax.

We could switch it to use the Dir syntax which removes the dependence on git

### PR DESCRIPTION
**Pull request by ochagata**
This is a PR to fix #235

### CODE REVIEW

File: napa.gemspec [hunk 1]
```diff
@@ -9,9 +9,9 @@ Gem::Specification.new do |gem|
   gem.homepage      = "https://tech.bellycard.com"
   gem.licenses      = ['MIT']
 
-  gem.files         = `git ls-files`.split($\)
+  gem.files         = Dir['**/*'].keep_if { |file| File.file?(file) }
   gem.executables   << 'napa'
-  gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
+  gem.test_files    = Dir['{test,spec,features}/**/*']
```
Comment (ochagata) [inline napa.gemspec]: I'll remove test and features since napa doesn't have these.
Will squash commits today and it should be ready to merge

**Comment by darbyfrey** [comment]
This is the default gemspec configuration that Bundler creates. What benefits do you see from removing the git dependency?

**Comment by ochagata** [comment]
@darbyfrey It makes the assumption that everyone uses git. Bundler basically enforces it because they use it in their `.gemspec`

**Comment by darbyfrey** [comment]
I see. Is that a poor assumption to make? Are people using something other than git these days? That's a real question - I'm curious if I'm out of the loop.

**Comment by ochagata** [comment]
While git is the industry standard, people still use subversion and perforce. I just say make it as agnostic to the user's configuration as possible.

**Comment by ochagata** [comment]
Also this change was merged in with #240

**Comment by darbyfrey** [comment]
Yea, that makes sense. :+1:

**Pull request closed without merging**